tracking-migration/src/test/java/io/customer/tracking/migration/testutils/utils/DateStub.kt

```diff
@@ -0,0 +1,32 @@
+package io.customer.tracking.migration.testutils.utils
+
+import io.customer.base.extenstions.getUnixTimestamp
+import io.mockk.every
+import io.mockk.mockkConstructor
+import io.mockk.unmockkConstructor
+import java.util.Date
+
+/**
+ * Stub class for [Date] object to provide a fixed date and facilitate testing.
+ * Call [mock] to mock the [Date] constructor and return the fixed date.
+ * Call [unmock] to unmock the [Date] constructor and return original date.
+ */
+class DateStub(val date: Date = Date()) {
```

Comment:
Because we use `Date()` directly in `JsonAdapter`. I don't think we need to add util just for migration `JsonAdapter`, so mocked `Date` using this. In case we have more usage of `Date()` in future, we can add util class to simplify it.